include/libdnf-cli/output/transaction_table.hpp

```diff
@@ -267,6 +317,9 @@ class TransactionSummary {
             case libdnf::transaction::TransactionItemAction::REASON_CHANGE:
                 reason_changes++;
                 break;
+            default:
```

Comment:
I think it would be better to explicitly list all actions here even if we are going to skip them with Todo.